### PR TITLE
[new_profile] can't create a new profile on major

### PR DESCRIPTION
**Pull request by kongtiaowang**
It needs to return a candID as a string, not an object.

### CODE REVIEW

File: modules/new_profile/php/new_profile.class.inc [hunk 1]
```diff
@@ -91,7 +91,7 @@ class New_Profile extends \NDB_Form
         /* Update front-end data. Use passed PSCID when present, otherwise
          * retrieve it from the newly-created candidate.
          */
-        $this->_candID = $candID;
+        $this->_candID = \Candidate::singleton($candID)->getCandID();
```
Comment (johnsaigle) [inline modules/new_profile/php/new_profile.class.inc]:
```suggestion
        $this->_candID = (string) $candID;
```

Comment (kongtiaowang) [reply modules/new_profile/php/new_profile.class.inc]: PHP can't swith object as a string.

**Comment by driusan** [comment]
@johnsaigle did you review this?

**Comment by johnsaigle** [comment]
I'll do a deeper review on Monday.

**Pull request merged**